corehq/apps/reports/standard/project_health.py

```diff
@@ -53,15 +52,17 @@ class MonthlyPerformanceSummary(jsonobject.JsonObject):
     active = jsonobject.IntegerProperty()
     performing = jsonobject.IntegerProperty()
 
-    def __init__(self, domain, month, users, has_filter,
+    def __init__(self, domain, month, users, has_filter, active_not_deleted_users,
```

Comment:
If so, you might be able to combine `users` and `has_filter` into something like `selected_users`:
```python
if selected_users:
    base_queryset = base_queryset.filter(user_id__in=selected_users)
```
(That change doesn't make sense in this PR, just raising it because it's a bit confusing)